vsphere/internal/virtualdevice/virtual_machine_network_interface_subresource_test.go

```diff
@@ -1,117 +0,0 @@
-package virtualdevice
```

Comment:
QQ - why was the test deleted vs extended?

Reply:
The unit test was testing a single function which I have removed.  I replaced the logic from nicUnitRange with something that in my mind was simpler, and works for both the non-sriov and sriov cases, where unit numbers either go from 7 descending or 45 descending.  Instead of using nicUnitRange to determine the length of array that needed to be populated, I created arrays of standard length, populate them and then remove any gaps, which simplifies the dealing with non-sriov/sriov cases.  
Example here: https://github.com/hashicorp/terraform-provider-vsphere/pull/1417/files#diff-e8de2d20cbf49f2db545eafc50b5230673e69274016a107bc7c75f6adcb1d635R594

As there was only one unit test and it was for a function that's gone, I removed the test too.